README.md

```diff
@@ -83,159 +55,112 @@ For more information on the importance of a professional README for your reposit
 </details>
 
 <details>
-  <summary>Server</summary>
+  <summary>CSS</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
+    <li><a href="#">CSS</a></li>
   </ul>
 </details>
 
 <details>
-<summary>Database</summary>
+<summary>Javascript</summary>
   <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="#">Javascript</a></li>
   </ul>
 </details>
 
-<!-- Features -->
-
 ### Key Features <a name="key-features"></a>
 
 > Describe between 1-3 key features of the application.
```

Comment:
- [ ] Please delete the template placeholder text: ` > Describe between 1-3 key features of the application.` in your readme, this will make your readme more professional 👍🏿